worlds/messenger/universal_tracker.py

```diff
@@ -0,0 +1,29 @@
+from Options import PlandoConnection
+from .connections import REVERSED_RANDOMIZED_CONNECTIONS
+from .portals import find_spot
+from .transitions import TRANSITIONS
+
+
+def reverse_portal_exists_into_portal_plando(portal_exists: dict) -> list[PlandoConnection]:
```

Comment:
```suggestion
def reverse_portal_exits_into_portal_plando(portal_exists: dict) -> list[PlandoConnection]:
```
and then all of the references to this